components/scream/src/control/tests/ad_tests.cpp

```diff
@@ -31,6 +31,16 @@ TEST_CASE ("group_requirements","[!throws]")
   // Init and run a single time step
   util::TimeStamp init_time(0,0,0,0.0);
   ad.initialize(atm_comm,ad_params,init_time);
+  // Verify that after initialization field "E" matches field "A"
+  auto& field_repo = ad.get_field_repo();
+  const auto& view_A = field_repo.get_field("A","Point Grid A").get_reshaped_view<const Real**, Host>();
+  const auto& view_E = field_repo.get_field("E","Point Grid A").get_reshaped_view<const Real**, Host>();
+  for (int icol=0;icol<num_cols;++icol) {
```

Comment:
I implemented `views_are_equal` in `field_utils.hpp` to do this kind of checks. But it was before we introduced subfields, so I think that routine needs to be upgraded.

Not for this PR though. But I opened an issue to track this.